src/popup.tsx

```diff
@@ -27,12 +28,15 @@ export const Popup = () => {
         Calculate CO2 emissions
       </button>
       <div >
-        SCI: {calculateCarbon(selectedCountries)} gCO2eq
+        Total Data Received: {totalBytesReceived} bytes
       </div>
       <div >
-        Specific Carbon Emissions (gC02 per byte): {averageSpecificEmissions}
+        Specific Carbon Emissions (grams of C02 per byte): {Math.round(averageSpecificEmissions * 100) / 100}
       </div>
```

Comment:
Can we change the `<divs>` to either a `<p>` or <h1/h2/h3 >` tag - this will help the app with accessibility because the correct roles will be applied

Reply:
Have replaced the div tags, but why does it help with accessibility?

Reply:
so the <p> and the heading tags implicitly come with aria-labels which are attached to the tags when rendered onto a page - this means that when a person uses a web page in an accessibility mode, they can clearly see that they are looking at a paragraph or a heading e.g. you can set up a screen reader w chrome for people with visual impairments which reads what type of content it os and says what the text is

Reply:
thanks!